Security/Risk_management/Rapid_Risk_Assessment.mediawiki

```diff
@@ -0,0 +1,356 @@
+reproducible, consistent and easy to communicate.
+
+This document assumes that the reader is familiar with risk definition, and the Mozilla risk levels from the
+[Risk_management] documentation (i.e. read it first).
+
+The Enterprise Information Security Team (Infosec) maintains this document as a reference guide for security engineers
+and anyone interested in following our risk management processes.
+
+Updates to this page should be submitted to the [https://github.com/mozilla/wikimo_opsec/ source repository on github].
+Changes are detailed in the [https://github.com/mozilla/wikimo_opsec/commits/master commit history].
+
+<span style="float: right; padding-top: 3em;">[[File:OpSec.png|300px]]</span>
+    </td>
+  </tr>
+</table>
+
+= Rapid Risk Assessment =
+
+{| class="wikitable"
+|-
+! <span style="color:red;">'''WARNING'''</span>
+|-
+|
+The Rapid Risk Analysis/Assessment (RRA) is not a security review or a full threat-model. A threat model, review, pen-testing, etc. may
+however follow an RRA.
+The RRA strongly focuses on getting the maximum amount of actionable information in the shortest amount of time available.
+'''The RRA process is made to analyze and assess services'''. Analyzing processes, or individual pieces of data is not
+recommended through the RRA process.
+|}
+
+== Preamble ==
+
+Data is the most important item in risk management. Software, websites, infrastructure, networks and people handle, process, exchange and store data.
+
+The RRA focuses efforts to create a summary of the risks associated with our data. In particular it aims to be:
+
+* '''Quick!''' About 30 minutes to 60 minutes maximum.
+* '''Very high-level'''. Details are for threat models.
+* '''Concise, readable'''. Short table with clear risk levels.
+* '''Re-runnable'''. Can be run during the architecture phase, and re-run if the project is re-architected or new high level design decisions are made.
+* '''Collects''' risk impact, an approximate impact likelihood and data dictionary.
+
+This helps to make the following type of decisions:
+
+* Can this project use systems which are easier to use but provide a lower security level?
+* How much should we care about maintenance, etc?
+* Should we create a full threat-model, pen-test the service, is privacy or legal team involvement necessary?
+* How long should we spend on these tasks?
+* Is there anything obvious I should really look at fixing right now?
+
+{| class="wikitable"
+|-
+! <span style="color:gray;">Sample use case</span>
+|-
+|
+Firefox accounts store user data:
+
+* What happens if that data is exposed to the world? What happens if the systems go down? What happens if that data is disclosed? What if the data is modified?
+* Are we at financial, operational risk? Is our reputation also at risk?
+* What kind of data does Firefox accounts exactly handle? How sensitive is the primary kind of data used by Firefox accounts?
+
+The RRA table has answers to all of these questions.
```

Comment:
The RRA table doesn't have the answers, but it has the questions we should be asking. It helps guide a risk discussion.